x/faucet/genesis.go

```diff
@@ -0,0 +1,33 @@
+package faucet
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ethermint/x/faucet/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// InitGenesis initializes genesis state based on exported genesis
+func InitGenesis(ctx sdk.Context, k Keeper, data types.GenesisState) []abci.ValidatorUpdate {
+	if acc := k.GetFaucetAccount(ctx); acc == nil {
+		panic(fmt.Sprintf("%s module account has not been set", ModuleName))
```

Comment:
will this affect the daemon running if it panics?

Reply:
it will panic on when starting the daemon (if the account is not set)